### ISSUE TITLE
Elevated circulating 3-hydroxyisovaleric acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6015]
**Preferred term label:** Elevated circulating 3-hydroxyisovaleric acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 3-hydroxyisovaleric acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating monocarboxylic acid cocentration HP:0010996

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think that 3-hydroxyisovaleric acid can be found to be elevated in the urine. I do not see mention of its elevation in the blood. I will add the corresponding urine term. Also a "plain" isovaleric acidemia term.
